src/data/party_menu.h

```diff
@@ -686,39 +686,41 @@ static const u16 sUnusedData[] =
     0x0121, 0x013b, 0x000f, 0x0013, 0x0039, 0x0046, 0x0094, 0x00f9, 0x007f, 0x0123,
 };
 
+static const u8 sText_Trade4[] = _("TRADE");
```

Comment:
I think removing the duplicate strings is a great idea! Things are kinda busy for me right now so I think it might be best if it's a follow-up PR so that it won't hold this PR up